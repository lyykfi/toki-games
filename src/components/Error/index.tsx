import React from 'react';
import { Paper, Typography } from '@material-ui/core';

interface ErrorProps {
	message: string;
}

const Error: React.FC<ErrorProps> = (props: ErrorProps): JSX.Element => {
	const { message } = props;

	return (
		<Paper>
			<Typography variant="h6" gutterBottom>
				{message}
			</Typography>
		</Paper>
	);
};

export default Error;
