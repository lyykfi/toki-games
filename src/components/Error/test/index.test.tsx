import React from 'react';
import { render } from '@testing-library/react';

import Error from '..';

describe('Error component', () => {
  it('Error component snapshot', async () => {
    const { container } = render(<Error message="test" />);

    expect(container).toMatchSnapshot();
  });
});
