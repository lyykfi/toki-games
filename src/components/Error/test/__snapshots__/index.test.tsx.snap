// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component Error component snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <h6
      class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom"
    >
      test
    </h6>
  </div>
</div>
`;
